src/functions/util/emojiExists.js

```diff
@@ -1,12 +1,28 @@
-module.exports = async d => {
+module.exports = async (d) => {
     const data = d.util.aoiFunc(d);
     if (data.err) return d.error(data.err);
 
-    const [emoji] = data.inside.splits;
+    let [emoji] = data.inside.splits;
 
-    data.result = !!(await d.util.getEmoji(d, emoji));
+    try {
+        emoji = d.util.getEmoji(d, emoji.addBrackets()).id;
+        data.result = true;
+    } catch {
+        emoji = emoji?.addBrackets() ?? undefined;
+    }
+
+    if (data.result === true) {
+        data.result = true;
+    } else {
+        if (!emoji) {
+            data.result = false;
+        } else {
+            const regex = /[\u{1F300}-\u{1F5FF}\u{1F900}-\u{1F9FF}\u{1FA70}-\u{1FAFF}\u{2600}-\u{27BF}\u{1F680}-\u{1F6FF}\u{24C2}-\u{1F251}\u{1F600}-\u{1F64F}]/u;
```

Comment:
`/<a?:.+?:\d{18}>|\p{Extended_Pictographic}/gu`

to match discord as well as unicode emoji

tho , test it

Reply:
Can't do this either, because if any match it'll just return true regardless if the emoji exists, therefore #getEmoji()

Reply:
u can do first getEmoji then unicodeemoji to match both discord and unicode emojis then

Reply:
thats what im doing though